.eslintrc.json

```diff
@@ -7,16 +7,52 @@
     "extends": [
         "eslint:recommended",
         "plugin:@typescript-eslint/recommended",
+        "plugin:tailwindcss/recommended",
         "prettier"
     ],
     "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module",
+        "tsconfigRootDir": ".",
+        "project": true
+    },
     "plugins": ["@typescript-eslint"],
     "rules": {
         "@typescript-eslint/no-unused-vars": [
             1,
             { "ignoreRestSiblings": true }
         ],
-        "@typescript-eslint/ban-ts-comment": "warn"
-    },
-    "root": true
+        "@typescript-eslint/ban-ts-comment": "warn",
+        "tailwindcss/classnames-order": [
```

Comment:
if anyone knows how to add a rule config like the "callees" you see here for _all_ the rules in a plugin instead of doing them individually like this please let me know